<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-H9B048JB" ignored="0" name="Test" tests="4" failures="1" timestamp="2020-11-26T18:14:38 CST" time="56.524" errors="0">
  <testcase name="validateToggleChecked" time="50.481" classname="Test.ToggleCheckOnTest"/>
  <testcase name="validateSingleToggleItemRemove" time="3.398" classname="Test.ToggleCheckOnTest"/>
  <testcase name="validateAllToggleItemRemove" time="1.412" classname="Test.ToggleCheckOnTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Executing Failed Test Method">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Executing Failed Test Method
at Test.ToggleCheckOnTest.validateAllToggleItemRemove(ToggleCheckOnTest.java:41)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validateAllToggleItemRemove -->
  <testcase name="tearDown" time="0.725" classname="Test.ToggleCheckOnTest"/>
</testsuite> <!-- Test -->
